lite/valor_lite/detection/manager.py

```diff
@@ -51,6 +57,74 @@
 """
 
 
+def _get_annotation_key(
```

Comment:
nit: could you rename this to `_get_valor_dict_annotation_key` since its only used in valor dict ingestion?